docs/source/howto/ssh.rst

```diff
@@ -196,7 +196,25 @@ This section explains how to use the ``proxy_command`` feature of ``ssh`` in ord
 SSH configuration
 ^^^^^^^^^^^^^^^^^
 
-Edit the ``~/.ssh/config`` file on the computer on which you installed AiiDA (or create it if missing) and add the following lines::
+To decide whether to use the ``ProxyJump`` (recommended) or the ``ProxyCommand`` directive, please check the version of your SSH client first with ``ssh -V``.
+The ``ProxyJump`` directive has been added in version 7.3 of OpenSSH, hence if you are using an older version of SSH (on your machine or the *PROXY*) you have to use ``ProxyCommand``.
+
+To setup the proxy configuration with ``ProxyJump`, edit the ``~/.ssh/config`` file on the computer on which you installed AiiDA (or create it if missing)
+and add the following lines::
+
+  Host SHORTNAME_TARGET
+      Hostname FULLHOSTNAME_TARGET
+      User USER_TARGET
+      IdentityFile ~/.ssh/aiida
+      ProxyJump USER_PROXY@FULLHOSTNAME_PROXY
+
+  Host FULLHOSTNAME_PROXY
```

Comment:
does one need that also for the proxycommand version below?

Reply:
for the proxycommand it would even be crucial since that configuration is not controlled by paramiko but only ssh (since the `ssh` process is spawned). Added.